docs/cicd/github-cd-template.yml

```diff
@@ -0,0 +1,67 @@
+# This is just an example workflow for continous deployment.
+# You should customize it to meet your own requirements.
+name: 'Continuous Deployment'
+on:
+  # When new commits are pushed onto the main branch.
+  push:
+    branches:
+      - main
+jobs:
+  buildAndPublish:
+    runs-on: ubuntu-latest
+    environment: test_environment
+    env:
+      AZURE_ACCOUNT_NAME: ${{secrets.AZURE_ACCOUNT_NAME}}
+      AZURE_ACCOUNT_PASSWORD: ${{secrets.AZURE_ACCOUNT_PASSWORD}}
+      AZURE_SUBSCRIPTION_ID: ${{secrets.AZURE_SUBSCRIPTION_ID}}
+      AZURE_TENANT_ID: ${{secrets.AZURE_TENANT_ID}}
+      M365_ACCOUNT_NAME: ${{secrets.M365_ACCOUNT_NAME}}
+      M365_ACCOUNT_PASSWORD: ${{secrets.M365_ACCOUNT_PASSWORD}}
+
+      # If the hosting environment is not provisioned, set this environment variable to true.
+      # or if it's provisioned and has not updates, set this environment variable to false.
+      RUN_PROVISION: true 
```

Comment:
would be good to prefix our env variables with something to namespace them to our team: `TFX_RUN_PROVISON` for example?

Reply:
or `TEAMSFX_RUN_PROVISION`

Reply:
Agree, I will change it to `TEAMSFX_RUN_PROVISION`.